tests/main/interfaces-kvm/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Ensure that the kvm interface works.
+
+details: |
+    The kvm interface allows read/write access to kvm.
+
+environment:
+    FAKE_KVM: "${PWD}/fake-kvm"
+    CONNECTED_PATTERN: ':kvm +test-snapd-kvm'
+    DISCONNECTED_PATTERN: '\- +test-snapd-kvm:kvm'
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-kvm
+
+restore: |
+    rm -f call.error
+    if [ -f $FAKE_KVM]; then
+        rm -f $FAKE_KVM
+        rm -f /dev/kvm
+    fi
+
+execute: |
+    echo "The interface is not connected by default"
+    snap interfaces | MATCH "$DISCONNECTED_PATTERN"
+
+    echo "When the interface is connected"
+    snap connect test-snapd-kvm:kvm
+    snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    if [ -c /dev/kvm ]; then
+        echo "Just testing the interface with fake kvm device"
```

Comment:
Maybe a comment like: `This test only works if there is no real /dev/kvm device already` ?

Reply:
Can we move the real `kvm` device aside and complete the test with a fake one? I don't see any reason not to.